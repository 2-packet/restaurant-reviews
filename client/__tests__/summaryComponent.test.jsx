import React from 'react';
import renderer from 'react-test-renderer';
import Summary from '../src/components/summary.jsx';

it('renders the summary sub-component', () => {
  const summary = {
    location: 'West Ernieville',
    noise: 'Average',
    recommendPercent: 36,
    valueRating: '4',
    averageOverall: '0.4',
    averageFood: '4.5',
    averageAmbience: '4.5',
    averageService: '5'
  };

  const tree = renderer
    .create(<Summary summary={summary} totalReviews={5} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
