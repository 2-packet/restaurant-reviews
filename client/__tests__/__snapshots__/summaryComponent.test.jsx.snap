// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the summary sub-component 1`] = `
<div
  id="reviews-summary"
>
  <div
    className="reviews-header"
  >
    What 
    5
     People Are Saying
  </div>
  <div
    id="reviews-summary-main"
  >
    <div
      id="reviews-summary-left-column"
    >
      <div
        className="reviews-subheader-bold"
      >
        Overall ratings and reviews
      </div>
      <div
        className="reviews-summary-text-large"
      >
        Reviews can only be made by diners who have eaten at this restaurant
      </div>
      <div
        id="reviews-summary-stars"
      >
        <div
          className="review-star"
        />
      </div>
      <div
        id="reviews-summary-average-ratings"
      >
        <span>
          0.4
           based on recent ratings
        </span>
      </div>
      <div
        id="reviews-summary-noise"
      >
        <div
          className="noise-icon"
        />
        <span>
           Noise Â· 
          Average
        </span>
      </div>
      <div
        id="reviews-summary-recommendation-percent"
      >
        <div
          id="thumb-icon"
        />
        <span>
          36
          % of people would recommend it to a friend.
        </span>
      </div>
      <a
        href="#"
        id="reviews-summary-best-link"
      >
        Best Restaurants in 
        West Ernieville
      </a>
    </div>
    <div
      id="reviews-summary-right-column"
    >
      <div
        id="reviews-rating-bars"
      >
        Bars go here
      </div>
    </div>
  </div>
</div>
`;
